datadog_checks_base/tests/base/utils/discovery/test_discovery.py

```diff
@@ -16,6 +16,20 @@ def test_include_empty():
     assert mock_get_items.call_count == 1
 
 
+def test_include_empty_exclude_non_empty():
+    mock_get_items = mock.Mock(return_value=['a', 'b', 'c', 'd', 'e', 'f', 'g'])
+    d = Discovery(mock_get_items, exclude=['b.*'])
+    assert list(d.get_items()) == []
+    assert mock_get_items.call_count == 1
```

Comment:
Would it make sense here to check the arguments to `mock_get_items`?

Reply:
I can! However the arguments to `mock_get_items` will always be empty- https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/utils/discovery/cache.py#L26

Reply:
All the more reason to assert something like
```python
assert mock_get_items.mock_calls == [mock.call()]
```

reference https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.mock_calls

Reply:
Sure- thoughts on replacing `assert mock_get_items.call_count == 1` with `mock_get_items.mock_calls == [mock.call()]` for every test? Or should we keep both?

Reply:
`mock_calls ==...` implies `call_count == 1`, so I'd replace everywhere.

Reply:
Agreed, updated!